ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -56,48 +46,46 @@ service UserManagementService {
 // Users and rights
 ///////////////////
 
-// Users are are used to manage the rights given to authenticated Ledger API clients.
+// Users are used to dynamically manage the rights given to Daml applications.
 // They are stored and managed per participant node.
 //
-// TLS client certificates authenticate the user specified by the certificates common name.
-// JWT tokens authenticate the user specified by the JWT tokens 'sub' (subject) claim.
+// Read the :doc:`Authorization documentation </app-dev/authorization>` to learn more.
 message User {
-    // The user identifier, which must be a non-empty string of at most 64
-    // characters matching the regexp [a-z0-9-_.], i.e. lower-case characters, digits,
-    // dashes or dots. Double-dots, double-dashes, and double-underscores are not allowed.
+    // The user identifier, which must be a non-empty string of at most 128
+    // characters that are either lowercase alphanumeric ASCII characters or one of the symbols "@^$.!`-#+'~_|".
     string id = 1;
 
     // The primary party as which this user reads and acts by default on the ledger
-    // _provided_ it has the corresponding ``CanReadAs(primary_party)`` or 
+    // *provided* it has the corresponding ``CanReadAs(primary_party)`` or
     // ``CanActAs(primary_party)`` rights.
-    //
     // Ledger API clients SHOULD set this field to a non-empty value for all users to
     // enable the users to act on the ledger using their own Daml party.
-    // Ledger API clients MAY set this field to empty for special users; e.g., a user
-    // that is granted ``CanReadAsAnyParty`` so that it can export the data for all parties
-    // hosted on the participant node.
+    // Users for participant administrators MAY not have an associated primary party.
```

Comment:
A little unclear - does this capitalization indicate 'must not' or 'may or may not'?

Reply:
may or may not

Reply:
fixed